test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -140,6 +140,17 @@ def test_create_payment_intent_with_setup_future_usage
     assert_equal 'on_session', response.params['setup_future_usage']
   end
 
+  def test_3ds_unauthenticated_authorize_with_off_session
+    options = {
+      currency: 'USD',
+      customer: @customer,
+      off_session: true,
+    }
+
+    assert response = @gateway.authorize(@amount, @three_ds_credit_card, options)
+    assert_failure response
```

Comment:
@sineed does it really test that `off_session`  param is used depending also on `confirm` values?  Seems like 2 separate specs are needed here verifying request body

Reply:
This was mostly a c/p of what was found for `off_session` in the main repo.
Btw why do we have to follow this c/p? Why not just to update gem as usual? What prevents from it?

Reply:
@sineed there is a lot of customization going on here as we were using this gateway probably before the PR was even approved on the active merchant's side and some interfaced has changed a bit since ten, which would require some adjustments on the app's side. Didn't have much time to figure this stuff out as I stopped when I saw a lot of various failures, but feel free to make it up-to-date.

BTW. some specs are failing.

Reply:
I see they're failing but they are not related to the changes.
Should I fix them in order to add this small change?

Reply:
@sineed Yeah, please do

Reply:
are you really sure I have to waste time fixing tests for gateways we don't use?

Reply:
I'm sure skipping unrelated tests should be enough